hash/risc0/bench/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Check if ZKBENCH_INPUT_SIZE_KB is set, otherwise set a default value
+ZKBENCH_INPUT_SIZE_KB=${ZKBENCH_INPUT_SIZE_KB:-"1"}  # Default to 1 if not set
+
+# Run benchmarks with the specified input size
+if [ "$ZKBENCH_INPUT_SIZE_KB" = "1" ]; then
+    cargo bench --bench bench_main -- 1
```

Comment:
Here too, instead of having cases, just pass a number as a command-line argument to the executable